gallery/l10n_cli/pubspec.yaml

```diff
@@ -1,11 +0,0 @@
-name: l10n_cli
```

Comment:
this file is still needed, no?

Reply:
or at least the dev_dependencies should be included in the gallery pubspec

Reply:
Nope, I moved the tool into the gallery/tool folder now so it’s using the packages that the rest of Gallery uses.

Reply:
And the dev dependencies were not used by the tool. It was only there because it was auto generated.

Reply:
Gotcha 👍